test/provision/k8s_install.sh

```diff
@@ -275,4 +275,7 @@ docker network create --subnet=192.168.9.0/24 outside
 docker run --net outside --ip 192.168.9.10 --restart=always -d docker.io/cilium/demo-httpd:latest
 docker run --net outside --ip 192.168.9.11 --restart=always -d docker.io/cilium/demo-httpd:latest
 
+#Preload the ip6tables table we need
+sudo modprobe ip6table_mangle
```

Comment:
Do we need this elsewhere (rather than just ginkgo CI provision)? Why isn't it autoloaded? Should we document it? Should we put it in the VM images in github.com/cilium/packer-ci-build? Container images?

Reply:
This will definitely need more discussion and a solution that works everywhere. It is not autoloaded because the cilium container has no privileges to do so (CAP_NET_ADMIN is not enough for kernel module autoloading).

Reply:
@jrajahalme does what you say mean that this may not work in existing Cilium setups upon upgrade?

Reply:
This issue needs still to be addressed. My preferred solution would be to give cilium container the prerequisite capability to (auto-) load kernel modules. @tgraf ??

Reply:
I'm not opposed to adding `CAP_SYS_MODULE` but we need to document why we need it and when it can be removed.